ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryRoutingFunctionalTest.java

```diff
@@ -376,10 +358,11 @@ public void shouldFilterLaggyServers() throws Exception {
 
     waitForRemoteServerToChangeStatus(clusterFormation.router.getApp(),
         clusterFormation.router.getHost(),
-        PullQueryRoutingFunctionalTest.lagsExist(clusterFormation.standBy.getHost(), 5));
+        HighAvailabilityTestUtil.lagsReported(3, clusterFormation.standBy.getHost(),
```

Comment:
This checks that the currentOffset is 3 and endOffset is 5, right?

Reply:
This checks that there are three lag reports.  It was hardcoded before.  currentOffset is Optional.empty() in this case.  Maybe I should change this method to ignore the number of reports since HighAvailabilityTestUtil.lagsReported(3) does that above.